qpc/messages.py

```diff
@@ -161,10 +161,19 @@
 
 REPORT_JSON_FILE_HELP = 'A list of files that contain the json details ' \
                         'reports to merge.'
+REPORT_JSON_DIR_HELP = 'The path to a directory that contain files of json ' \
+                       'details reports to merge'
 REPORT_JSON_FILES_HELP = 'At least two json details report files are ' \
                          'required to merge.'
 REPORT_INVALID_JSON_FILE = 'The file %s does not contain a valid json ' \
                            'details report.'
+REPORT_JSON_DIR_NO_FILES = \
+    'No files with extension .json found in the provided path.'
+REPORT_JSON_DIR_CHECK_FILES = \
+    'Checking files for the correct structure. %s'
```

Comment:
Change this to say something like `Checking files for valid json details report.`